lib/inputs/os-release/static.ts

```diff
@@ -0,0 +1,92 @@
+import { getContentIfExists } from "../../extractor";
+import { ExtractAction, ExtractedLayers } from "../../extractor/types";
+import { streamToString } from "../../stream-utils";
+
+export { getOsReleaseActions, getOsRelease };
+
+const getOsReleaseAction: ExtractAction = {
+  actionName: "os-release",
+  fileNamePattern: "/etc/os-release",
+  callback: (stream) => streamToString(stream),
+};
+
+const getLsbReleaseAction: ExtractAction = {
+  actionName: "lsb-release",
+  fileNamePattern: "/etc/lsb-release",
+  callback: (stream) => streamToString(stream),
+};
+
+const getDebianVersionAction: ExtractAction = {
+  actionName: "debian-version",
+  fileNamePattern: "/etc/debian_version",
+  callback: (stream) => streamToString(stream),
+};
+
+const getAlpineReleaseAction: ExtractAction = {
+  actionName: "alpine-release",
+  fileNamePattern: "/etc/alpine-release",
+  callback: (stream) => streamToString(stream),
+};
+
+const getRedHatReleaseAction: ExtractAction = {
+  actionName: "redhat-release",
+  fileNamePattern: "/etc/redhat-release",
+  callback: (stream) => streamToString(stream),
+};
+
+const getOracleReleaseAction: ExtractAction = {
+  actionName: "oracle-release",
+  fileNamePattern: "/etc/oracle-release",
+  callback: (stream) => streamToString(stream),
+};
+
+const getOsReleaseActions: ExtractAction[] = [
+  getOsReleaseAction,
+  getLsbReleaseAction,
+  getDebianVersionAction,
+  getAlpineReleaseAction,
+  getRedHatReleaseAction,
+  getOracleReleaseAction,
+];
+
+function getOsRelease(extractedLayers: ExtractedLayers): string {
+  const osRelease = getContentIfExists(extractedLayers, getOsReleaseAction);
+  return osRelease !== undefined ? (osRelease as string) : "";
+}
+
+export function getLsbRelease(extractedLayers: ExtractedLayers): string {
+  const lsbRelease = getContentIfExists(extractedLayers, getLsbReleaseAction);
+  return lsbRelease !== undefined ? (lsbRelease as string) : "";
+}
+
+export function getDebianVersion(extractedLayers: ExtractedLayers): string {
```

Comment:
shouldn't the function name be `getDebianRelease`? as `getAlpineRelease`, `getRedHatRelease`, etc?

Reply:
I've kept the names as they exist right now, this change would need some input from the folks who did it like this the first time 🤔

Reply:
np 👍 let's not resolve it right now and see what @jurglic think :)?

Reply:
Yeah, makes more sense to me to call it `getDebianRelease`.. unfortunatelly don't know why was named like this in first place, maybe just because of rush? :)

Reply:
So the file we're inspecting is `/etc/debian_version` so maybe that's why it's called like that? I think I'll keep it so that it's consistent...